.rustfmt.toml

```diff
@@ -5,7 +5,7 @@ format_strings = true
 match_block_trailing_comma = true
 max_width = 80
 merge_derives = true
-merge_imports = true
+imports_granularity = "Crate"
```

Comment:
What's the difference between these two? I'm not too familiar with rustfmt config

Reply:
this was the suggested resolution to a cli warning